### ISSUE TITLE
Cannot run builtin nodejs module like `util/types`

### ISSUE THREAD

**Issue by axmad386** [oven-sh/bun#323]
Hi, Just try bun and this is amazing. It's super fast. But I have problem to run nodejs builtin module like `util/types`. When I run script that import it, bun throw error.
```sh
error: Could not resolve: "util/types". Maybe you need to "bun install"?
```
I try to install `util` but still fail. `util/types` also not available on npm. How to solve this problem? 
Thank you

**Comment by FinnRG** [comment]
The util package should be injected by bun. Could you provide a minimal reproducible example?

**Comment by axmad386** [comment]
@FinnRG Hi, I found strange behaviour. I think the problem is we only can import `util` but not `util/types`. So this example will throw error
```js
// this work on node but fail in bun
import {isProxy} from "util/types"
isProxy(something)
```

```js
// And.. this is work on both node and bun
import {types} from "util"
types.isProxy(something)
```
But I think it will better if we can import only part of util like node do

**Comment by axmad386** [comment]
@FinnRG just want to confirm. Is `util/types` is not exported yet by bun? 
If only `util` that is exported by bun, I will refactor my code to this and I will close this issue
```js
import {types} from "util"
```

**Comment by paperclover** [comment]
This is fixed as of 0.6.7
